tests/integration/security/authorization_test.go

```diff
@@ -618,16 +616,17 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.egress-gateway").
 		Run(func(t framework.TestContext) {
-			ns := namespace.NewOrFail(t, t, namespace.Config{
-				Prefix: "v1beta1-egress-gateway",
-				Inject: true,
-			})
-
-			var a, b, c echo.Instance
+			ns := apps.Namespace1
+			rootns := newRootNS(t)
+			a := apps.A.Match(echo.Namespace(apps.Namespace1.Name()))
+			b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+			vm := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
+			// Set up workload X that is accessible via egress gateway.
+			var x echo.Instance
+			XSvc := "x"
 			echoboot.NewBuilder(t).
-				With(&a, util.EchoConfig("a", ns, false, nil)).
-				With(&b, echo.Config{
-					Service:   "b",
+				With(&x, echo.Config{
```

Comment:
nit: we'd like to deprecate` and use `WithConfig` + `Match` instead to get rid of passing a pointer for assignment.

Reply:
Thanks.